src/middlewares/auth.middlewares.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response, NextFunction } from 'express';
+import { verifyAccessToken } from '@modules/jwt';
+import { fail } from '@modules/response';
+import statusCode from '@modules/statusCode';
+import message from '@modules/message';
+
+const authJWT = (req: Request, res: Response, next: NextFunction) => {
+  let token: string = '';
+  if (req.headers['authorization']) {
+    token = req.headers['authorization']!.split('Bearer ').reverse()[0];
+  }
+
+  if (!token) {
+    return res
+      .status(statusCode.UNAUTHORIZED)
+      .send(fail(statusCode.UNAUTHORIZED, message.NULL_VALUE_ACEESS_TOKEN));
+  }
+
+  const payload = verifyAccessToken(token);
+
+  if (payload.ok) {
+    req.body.userId = payload.userId;
```

Comment:
```suggestion
    req.userId = payload.userId;
```
req에 담는 게 좋을 것 같습니다.